service/src/main/kotlin/fi/espoo/evaka/pis/controllers/ParentshipController.kt

```diff
@@ -69,6 +69,7 @@ class ParentshipController(
         Audit.ParentShipsCreate.log(
             targetId = AuditId(listOf(body.headOfChildId, body.childId)),
             objectId = AuditId(parentship.id),
+            meta = mapOf("modifiedBy" to user.evakaUserId),
```

Comment:
Meillä menee jo jokaisesta requestista automaattisesti käyttäjätieto lokeille, joten näitä ei tarvi laittaa käsin